src/shackle_server.erl

```diff
@@ -271,6 +271,6 @@ terminate(Reason, #state {
 
     cancel_timer(TimerRef),
     ok = Client:terminate(ClientState),
```

Comment:
Remove the dynamic function call on line 273. Only modules that define callbacks should make dynamic calls.